src/components/inputs/rich-text-input/rich-text-input.js

```diff
@@ -1,25 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
 import requiredIf from 'react-required-if';
-import Types from 'slate-prop-types';
 import { Editor } from 'slate-react';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
 import renderEditor from './editor';
 import plugins from '../../internals/rich-text-plugins';
 import html from '../../internals/rich-text-utils/html';
 import isEmpty from '../../internals/rich-text-utils/is-empty';
 
-class RichTextInput extends React.PureComponent {
+const removeProps = ['value'];
+
+class RichTextInput extends React.Component {
+  state = {
+    // we keep track of the serialized (HTML) value
+    serializedValue: this.props.value || '',
+    value: html.deserialize(this.props.value || ''),
+  };
+
+  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
instead of using PureComponent, we do something similar but ignoring the prop 'value'

Reply:
I like this approach, but I've tested and found that we're re-rendering the RichTextInput a lot more times than in the previous "version".
For instance, I noticed that when out of focus, clicking inside the input causes 9 (!) more renders, without actually typing anything.

Haven't tracked down the cause yet 🤔

Reply:
have you tried pulling the latest code?

Reply:
Yes, I'm testing with the latest.

Reply:
Yeah the problem is that `ReactValue`'s onChange is changing on every re-render.